lisa/util/parallel.py

```diff
@@ -25,7 +29,9 @@ def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> Optional[bool]:
 
     def submit_task(self, task: Callable[[], T_RESULT]) -> None:
         future: Future[T_RESULT] = self._pool.submit(task)
+        self._future_task_map[future] = task
         self._futures.append(future)
+        self._log.debug(f"Started task : {task}")
```

Comment:
It's duplicated with the line in runner, so the line in runner should be removed. And the task may be too long, so add the truncation here.

Reply:
The line in runner also has `runner.id` which started this task.  Do you think runner.id would be a useful debugging information?

Reply:
The main reason is the description of test case may be too long. But I don't want to handle it specially here. Maybe we can keep 300 chars?